openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Login.spec.ts

```diff
@@ -140,7 +140,7 @@ test.describe('Login flow should work properly', () => {
     await page.locator('[data-testid="go-back-button"]').click();
   });
 
-  test.skip('Refresh should work', async ({ browser }) => {
+  test.fixme('Refresh should work', async ({ browser }) => {
```

Comment:
Did we not fix this one?

Reply:
Not it's not yet fixed hence added fix me working on it sepratly